src/guacd/daemon.c

```diff
@@ -320,6 +321,14 @@ int main(int argc, char* argv[]) {
     /* General */
     int retval;
 
+#ifdef HAVE_DECL_PTHREAD_SETATTR_DEFAULT_NP
+    /* Set default stack size of 8MB */
```

Comment:
I don't love having the value of the constant specified here in the constant - it's an opportunity for things to get out of sync if somebody later updates the value of `GUACD_THREAD_STACK_SIZE`.

Reply:
Makes sense. I've removed that part of the comment.